DNN Platform/DotNetNuke.ContentSecurityPolicy/ContentSecurityPolicy.cs

```diff
@@ -0,0 +1,479 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.ContentSecurityPolicy
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// Manages the entire Content Security Policy.
+    /// </summary>
+    public class ContentSecurityPolicy : IContentSecurityPolicy
+    {
+        private string nonce;
+
+        /// <summary>Initializes a new instance of the <see cref="ContentSecurityPolicy"/> class.</summary>
+        public ContentSecurityPolicy()
```

Comment:
Given that we don't need to do anything special here with the constructor, this should be omitted